scheduler/src/cook/kubernetes/api.clj

```diff
@@ -416,9 +443,15 @@
                                                                  .getResources
                                                                  .getRequests
                                                                  convert-resource-map))
-                                                           containers)]
-                               (apply merge-with + container-requests))))
-                      (apply merge-with +)))
+                                                           containers)
+                                   {:keys [gpus] :as resource-map} (apply merge-with + container-requests)
+                                   gpu-model (-> pod .getSpec .getNodeSelector (get "cloud.google.com/gke-accelerator"))]
+                               (if (and gpu-model (pos? gpus))
+                                 (assoc resource-map :gpus {gpu-model gpus})
+                                 (assoc resource-map :gpus {})))))
+                       ;; sum-resources will need to handle the unlikely case where
```

Comment:
Maybe put in a comment that 'We assume and require that a node will not have more than one GPU type"